features/dom.yml

```diff
@@ -92,6 +92,7 @@ compat_features:
   - api.Document.createTreeWalker.whatToShow_filter_parameters_optional
   - api.Document.currentScript
   - api.Document.defaultView
+  - api.Document.designMode
```

Comment:
Googling "designMode" had me thinking this might almost be a feature by itself, albeit a novelty. 😄

https://www.reddit.com/r/webdev/comments/npvmo0/that_feeling_when_you_first_discovered/

(It's fine to leave this here.)

Reply:
yeah there's an alternate timeline where this got big in authoring tools...